lib/Hoymiles/src/parser/AlarmLogParser.cpp

```diff
@@ -113,17 +113,17 @@ const std::array<const AlarmMessage_t, ALARM_MSG_COUNT> AlarmLogParser::_alarmMe
     { AlarmMessageType_t::HMT, 221, "Abnormal wiring of grid neutral line", "", "" },
     { AlarmMessageType_t::ALL, 222, "PV-4: Input undervoltage", "PV-4: Eingangsunterspannung", "PV-4: Sous-tension d’entrée" },
 
-    { AlarmMessageType_t::ALL, 301, "FB-A: internal short circuit failure", "", "" },
-    { AlarmMessageType_t::ALL, 302, "FB-B: internal short circuit failure", "", "" },
+    { AlarmMessageType_t::ALL, 301, "FB-A: internal short circuit failure", "FB-A: Interner Kurzschlussfehler", "" },
+    { AlarmMessageType_t::ALL, 302, "FB-B: internal short circuit failure", "FB-B: Interner Kurzschlussfehler", "" },
 
-    { AlarmMessageType_t::ALL, 303, "FB-A: overcurrent protection failure", "", "" },
-    { AlarmMessageType_t::ALL, 304, "FB-B: overcurrent protection failure", "", "" },
+    { AlarmMessageType_t::ALL, 303, "FB-A: overcurrent protection failure", "FB-A: Überstromschutzfehler", "" },
+    { AlarmMessageType_t::ALL, 304, "FB-B: overcurrent protection failure", "FB-B: Überstromschutzfehler", "" },
 
-    { AlarmMessageType_t::ALL, 305, "FB-A: clamp circuit failure", "", "" },
-    { AlarmMessageType_t::ALL, 306, "FB-B: clamp circuit failure", "", "" },
+    { AlarmMessageType_t::ALL, 305, "FB-A: clamp circuit failure", "FB-A: Fehler im Klemmkreis", "" },
+    { AlarmMessageType_t::ALL, 306, "FB-B: clamp circuit failure", "FB-B: Fehler im Klemmkreis", "" },
 
-    { AlarmMessageType_t::ALL, 307, "INV power device failure", "", "" },
-    { AlarmMessageType_t::ALL, 308, "INV overcurrent or overvoltage protection failure", "", "" },
+    { AlarmMessageType_t::ALL, 307, "INV power device failure", "Ausfall des Wechselrichter-Leistungsgeräts", "" },
```

Comment:
Wouldn't "Ausfall des Wechselrichters" be enough?

Reply:
How do you find: "Ausfall der Stromversorgung des Wechselrichters"

Reply:
That would depend on what actually failed in this case? Because the "Stromversorgung des Wechselrichters" would actually be the solar panels, as those feed the power to the inverter right? I think I read somewhere that the inverter itself never draws its power from the grid itself. Or is it the grid failing in this case?

Reply:
Since it is a hardware error and this only occurs rarely, I can also remove the translation.

Reply:
The translation as “Stromversorgung” is definitely wrong. Here is another spelling "Wechselrichter – Ausfall des Leistungsgeräts".